src/client/apps/settings/client/curations/gucci/test/metadata.test.tsx

```diff
@@ -155,7 +155,7 @@ describe("Metadata", () => {
         value: "Email Tags",
       },
     } as unknown) as React.FormEvent<HTMLInputElement>
-    input.onChange(event)
+    input.onChange && input.onChange(event)
```

Comment:
Only q so far is why the need for these guards?

Reply:
They raise a type error than input is possibly undefined. This is the quickest fix, but it is on my TODO's to change these components to palette's input, at which time will go back to these tests.